cmd/swarmctl/node/common.go

```diff
@@ -138,6 +138,16 @@ func getNode(ctx context.Context, c api.ControlClient, input string) (*api.Node,
 		}
 
 		if l := len(rl.Nodes); l > 1 {
+			for _, n := range rl.Nodes {
+				name := n.Spec.Annotations.Name
```

Comment:
@jinuxstyle This is an oversight. User should be able to set a name on a node and have it work correctly.